pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -1578,8 +1602,8 @@ func (c *client) ScalePacificClusterControlPlane(clusterName, namespace, apiVers
 	return nil
 }
 
-func (c *client) ScalePacificClusterWorkerNodes(clusterName, namespace, apiVersion string, workersCount int32) error {
-	tkcObj, err := c.getPacificClusterObject(clusterName, namespace, apiVersion)
+func (c *client) ScalePacificClusterWorkerNodes(clusterName, namespace string, workersCount int32) error {
+	tkcObj, err := c.getPacificClusterObject(clusterName, namespace)
```

Comment:
Is the plan to revisit the scale logic with a different PR?

Reply:
Yes. The plan is to have seperate PR for scaling logic for TKGS